Packs/Salesforce/Scripts/SalesforceAskUser/SalesforceAskUser.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+retries = int(demisto.args().get('retries'))
```

Comment:
please use args variable at the beginning of the script and use it in every other place in the script. 
e.g.:
```args = demisto.args()```